webpack/assets/javascripts/react_app/components/HostDetails/Templates/CardItem/CardTemplate/index.js

```diff
@@ -4,33 +4,48 @@ import {
   Card,
   CardActions,
   CardHeader,
+  CardExpandableContent,
   Dropdown,
   KebabToggle,
   CardTitle,
   CardBody,
   GridItem,
 } from '@patternfly/react-core';
 
-const CardItem = ({
+import { useLocalStorage } from '../../../../../common/hooks/Storage';
+import { useDidUpdateEffect } from '../../../../../common/hooks/Common';
+
+const CardTemplate = ({
   header,
   children,
+  expandable,
+  isExpandedGlobal,
   dropdownItems,
   overrideGridProps,
   overrideDropdownProps,
 }) => {
   const [dropdownVisibility, setDropdownVisibility] = useState(false);
+  const [isExpanded, setExpanded] = useLocalStorage(
+    `${header} card expanded`,
+    true
+  );
   const onDropdownToggle = isOpen => setDropdownVisibility(isOpen);
-
+  const onExpandCallback = () => setExpanded(prevState => !prevState);
   const onDropdownSelect = event => {
     setDropdownVisibility(prevState => !prevState);
     // https://github.com/eslint/eslint/issues/12822
     // eslint-disable-next-line no-unused-expressions
     overrideDropdownProps?.onSelect?.(event);
   };
+  useDidUpdateEffect(
+    () => isExpandedGlobal !== undefined && setExpanded(isExpandedGlobal),
+    [isExpandedGlobal]
+  );
+
   return (
     <GridItem xl2={3} xl={4} md={6} lg={4} {...overrideGridProps}>
-      <Card isHoverable>
-        <CardHeader>
+      <Card isExpanded={isExpanded} isSelectable>
```

Comment:
It looks like `isSelectable` makes the cursor turn into a hand (like you're hovering on a link) for the entire card. Is this desired? :confused:

Reply:
Nope :) we wanted to remove it in a follow up pr, the change here from `isHoverable` is because `isHoverable` was deprecated